sim/plotscript.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy
+import glob
+import os
+from pathlib import Path
+import shutil
+
+
+dirs = glob.glob("/home/cameronnicholls/boxplots/OFAT/*/stats.csv")
```

Comment:
I think this should be a function in `ofat.py` called `plot`

Ideally the function would take a list of preloaded dataframes as input. For starters though, can just be a path to the root of the ofat directory.

Reply:
Hey just to clarify, by this do you mean the whole script could be replaced with a function in `ofat.py` ?

Reply:
yes, easier for everyone to use if it's  abstracted into a function. I think `ofat.py` is a sensible module to put it in for now.